webapp/fmprod/src/main/java/org/apache/oodt/cas/product/jaxrs/resources/ProductPageResource.java

```diff
@@ -98,6 +100,10 @@ public ProductPageResource(
     }
   }
 
+  public void setTotalProducts(int totalProducts){
```

Comment:
Minor: lets add java doc comment here and getter

Reply:
added